charts/ztka/templates/_helpers.tpl

```diff
@@ -103,6 +103,8 @@ Get DB Address.
 {{- define "ztka.dbAddr" -}}
   {{- if .Values.deploy.postgresql.enable -}}
 {{.Release.Name}}-postgresql.{{.Release.Namespace}}.svc.cluster.local
+  {{- else if .Values.deploy.postgresql.existingSecret -}}
+{{- printf "%s" (tpl .Values.deploy.postgresql.existingSecret $) -}}
```

Comment:
This is returning existing secret name, but instead we are expecting db address.

Need to fetch db address from k8s secret.